src/network/networkd-network-gperf.gperf

```diff
@@ -612,6 +613,7 @@ DHCP.UseDomainName,                          config_parse_use_domains,
 DHCP.UseRoutes,                              config_parse_bool,                                        0,                             offsetof(Network, dhcp_use_routes)
 DHCP.Anonymize,                              config_parse_bool,                                        0,                             offsetof(Network, dhcp_anonymize)
 DHCP.SendHostname,                           config_parse_dhcp_send_hostname,                          AF_UNSPEC,                     0
+DHCP.Bootp,                                  config_parse_bool,                                        0,                             offsetof(Network, dhcp_send_bootp)
```

Comment:
[DHCP] section is kept for backward compatibility. Please do not add a new setting here.

Reply:
I suspected as much. I'll remove.